ada_2_fhir-r4/fhir/2_fhir_fhir_include.xsl

```diff
@@ -201,39 +202,67 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
         </xsl:choose>
     </xsl:function>
     
-    <!-- Outputs reference if input is ADA or fhirMetadata element -->
+    <!-- Outputs reference if input is ADA, fhirMetadata or ADA reference element -->
     <xsl:template name="makeReference">
-        <xsl:param name="in" select="." as="element()*"/>
+        <xsl:param name="in" select="." as="element()*"/> <!-- Could be an ADA instance or an ADA reference element -->
         <xsl:param name="elementName" as="xs:string" required="yes"/>
+        <xsl:param name="wrapIn" as="xs:string?"/>
```

Comment:
Dit is een mechanisme om het volledige element te genereren wanneer er een referentie is en om het element weg te laten als de referentie er niet is. Dat maakt de aanroep makkelijker.

Reply:
Dit is al wel eerder toegepast (weet even niet meer waar), maar dit is specifiek relevant voor situaties waarvan je niet weet óf er output komt. Dan krijg je de ietwat omslachtige situatie dat je eerst een test moet doen of er output gegenereerd wordt en en zo ja, dan pas het element gaan maken.